src/amo/components/RatingManager/styles.scss

```diff
@@ -30,4 +23,10 @@ $button-height: $rating-l-star-size + $rating-l-star-gutter -
   legend {
     word-wrap: break-word;
   }
+
+  .Rating {
+    @include respond-to(large) {
+      margin: 12px;
```

Comment:
maybe we could keep this margin at smaller viewports too? At least on top:

![Alt text](https://monosnap.com/image/c3Ux0Pyuh9y2Wutn7yimxpjBCQj7rE.png)

what do you think?

Reply:
Good idea, thanks.